docs/source/_inc/_branch-policy.md

```diff
@@ -31,11 +31,17 @@ canary
 
     If the pull request is a feature or a bugfix, and if the release manager deems it useful to the latest version's branch, they may ask you to backport it to that branch.
 
-`17.x.x`
-:   `17.x.x` is the current stable and actively developed branch in git.
-    Upon the final release of version `18.0.0`, the `17.x.x` branch line will be no longer supported.
+`18.x.x`
+:   `18.x.x` is the current stable and actively developed branch in git.
+    This version of Volto has the [same maintenance and support schedule as Plone 6.1](https://plone.org/download/release-schedule).
     Any new feature will be merged into the `main` branch, and only backported to and released in older versions, if the Volto Team approves it.
 
+`17.x.x`
+:   `17.x.x` is no longer supported and became legacy when Volto 18 was released.
+    It won't receive any bug fixes nor security updates.
+    It is recommended that you update to 18.x.x.
+    If you need a bug fix or security update for this version, please submit a pull request, and the Volto Team will review it to determine whether it's suitable to merge.
```

Comment:
This seems to contradict line 41.

Reply:
I'm adding a "legacy" entry at the top of the document to clarify it.

Reply:
We shouldn't tell people to submit security issues or pull requests publicly. It should go through security@plone.org, unless I'm totally off about that. True story, I once got slapped for creating an issue stating that a Plone repo's primary branch lacked branch protection, even though it was stated publicly on the repo's home page.

Also I think we should tell people to create an issue to discuss a bug fix with the Volto Team before doing any work with a pull request.

```suggestion
    If you need a bug fix, please create an issue to discuss with the Volto Team.
    For security issues, please contact the Plone Security Team by sending email to security@plone.org.
```